src/main/java/ch/njol/skript/events/EvtBlock.java

```diff
@@ -144,8 +164,8 @@ else if (o instanceof BlockData && finalBlockData != null)
 	}
 	
 	@Override
-	public String toString(final @Nullable Event e, final boolean debug) {
-		return "break/place/burn/fade/form of " + Classes.toString(types);
+	public @NotNull String toString(@Nullable Event event, boolean debug) {
```

Comment:
This `@NotNull` isn't needed at this position when Skript is the one to call this, and Skript already asserts it not null.
Same with import for this annotation.